dali/operators/image/crop/crop_mirror_normalize.cc

```diff
@@ -53,7 +53,10 @@ normalization only.
   .AddOptionalArg("std",
     R"code(Standard deviation values for image normalization.)code",
     std::vector<float>{1.0f})
-  .AddParent("Crop");
+  .AddOptionalArg("image_type",
+    R"code(The color space of input and output image)code",
```

Comment:
Do you need that for crop?

Reply:
I don't. I deprecated it in a separate PR: https://github.com/NVIDIA/DALI/pull/2061